### DISCUSSION TITLE
Are there standard open code to compare the results in papers?

### DISCUSSION THREAD

**Discussion by finger1517** [facebookresearch/fastMRI#204] [Q&A]
in many images restoration papers and image super resolution papers,
the images showed like below, are there any code provided in python or matlab there to do the ZOOM IN job?
I am a rookie in this area, so I would like to thank you all for giving advice or responces
![image](https://user-images.githubusercontent.com/33984530/148698383-741225cf-e9e3-4c2d-9eb4-6314e7245b87.png)

![image](https://user-images.githubusercontent.com/33984530/148698407-5b38ae52-a1a6-4f5b-bc85-1fcb00fad858.png)

**Comment by mmuckley** [comment]
When I have made figures like these I think I have used some document editing software like LaTeX with [tikz](https://www.overleaf.com/learn/latex/TikZ_package).
